app/assess/routes.py

```diff
@@ -30,12 +34,61 @@ def funds():
     from fund store
     :return:
     """
-
     funds = get_funds()
-
     return render_template("funds.html", funds=funds)
 
 
+@login_requested
+@assess_bp.route(
+    "score/application_id/<application_id>/sub_criteria_id/<sub_criteria_id>",
+    methods=["POST", "GET"],
+)
+def application_sub_crit_scoring(application_id: str, sub_criteria_id: str):
+    fund = get_fund(Config.COF_FUND_ID)
+    form = JustScoreForm()
+    if form.validate_on_submit():
+        score = int(form.score.data)
+        justification = form.justification.data
+        try:
+            user_id = g.account_id
+        except AttributeError:
+            user_id = (  # TODO remove and force g.account_id after adding authentication # noqa
+                ""
+            )
+        submit_score_and_justification(
+            score=score,
+            justification=justification,
+            application_id=application_id,
+            user_id=user_id,
+            timestamp=datetime.now().strftime("%Y/%m/%d %H:%M:%S"),
```

Comment:
Times should be in UTC and ISO format, so I think  `datetime.utcnow().isoformat()` is what you want here, if you can test if that works with the assessment-store?

Reply:
this is getting auto-created from assessment store now so will remove this from here completely  👍🏾